arcgis-android-toolkit/src/main/java/com/esri/arcgisruntime/toolkit/ar/ArcGISArView.kt

```diff
@@ -627,7 +630,13 @@ class ArcGISArView : FrameLayout, DefaultLifecycleObserver, Scene.OnUpdateListen
      * @since 100.6.0
      */
     fun stopTracking() {
-        arSceneView?.pause()
+        arSceneView?.let {
+            it.scene?.let { scene ->
```

Comment:
could this become a one liner like you did above where you add the listener?

```
arSceneView?.scene?.let { }
```

Reply:
Afraid, because I need the `arSceneView` to `pause()`. If I wrapped it like you've suggested I'd not be able to access the `arSceneView`.

Reply:
Gotcha, makes sense.